src/node_process_methods.cc

```diff
@@ -157,14 +157,20 @@ static void Kill(const FunctionCallbackInfo<Value>& args) {
   args.GetReturnValue().Set(err);
 }
 
-static void MemoryUsage(const FunctionCallbackInfo<Value>& args) {
+static void Rss(const FunctionCallbackInfo<Value>& args) {
   Environment* env = Environment::GetCurrent(args);
 
   size_t rss;
   int err = uv_resident_set_memory(&rss);
   if (err)
     return env->ThrowUVException(err, "uv_resident_set_memory");
 
+  args.GetReturnValue().Set(static_cast<uint32_t>(rss));
```

Comment:
Why a cast to `uint32_t`? Isn't it possible that RSS is > 4GB on a 64Bit OS?
in `MemoryUsage` a `double` is used.